### PR TITLE
Switch matomo opt-out to ga opt-out

### PR DESCRIPTION
**Pull request by Bitcoinera**
Could not test this on localhost. No GA there

### CODE REVIEW

File: src/app/components/Footer/index.tsx [hunk 1]
```diff
@@ -79,13 +79,15 @@ export function Footer() {
...
+        {hasGA && (
           <div className="d-flex flex-row justify-content-between align-items-center text-lightGrey mt-5">
-            <iframe
-              title="MatomoOptout"
-              style={{ width: '100%', border: 'none', marginLeft: '-5px' }}
-              src="https://sovrynapp.matomo.cloud/index.php?module=CoreAdminHome&action=optOut&language=en&backgroundColor=171717&fontColor=ffffff&fontSize=14px&fontFamily=system-ui"
-            />
+            <a
+              title="GoogleAnalyticsOptout"
+              target="_blank"
```
Comment (creed-victor) [inline src/app/components/Footer/index.tsx:86]:
```suggestion
              target="_blank"
              rel="noopener noreferrer"
```

**Comment by Bitcoinera** [comment]
This is how it is looking now. Probably should try to make it look fancier

image

**Comment by Bitcoinera** [comment]
So it looks like right now

image

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -180,6 +180,7 @@
     "notice_2": "Sempre verifique as taxas de transação antes de realizar operações na SOVRYN. Algumas carteiras podem configurar automaticamente taxas muito altas.",
     "notice_3": "Visite nosso <0>FAQ</0> se tiver dúvidas.",
     "notice_4": "Se você é um usuário novato, veja o <0>wiki</0> e aprenda a usar a SOVRYN.",
+    "optOut": "Deixar de participar no <0>Google Analytics</0>",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:183]:
```suggestion
    "optOut": "Desativar o <0>Google Analytics</0>",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change PT-BR

**Review by creed-victor — APPROVED** [review]

**Pull request merged**